pyas2/models.py

```diff
@@ -19,6 +19,10 @@
 from pyas2.utils import run_post_send
 from pyas2.utils import store_file
 
+from pyas2lib.utils import pem_to_der
+from OpenSSL import crypto
+from datetime import datetime
+
 
 class PrivateKey(models.Model):
```

Comment:
Can we extract the info for private keys as well?

Reply:
Okay move this to utils and add test case for this function, one for private and another for public key...

Reply:
Moved, please check PR on pyas2-lib. New release of pyas2-lib required for this to be finalized.

Reply:
I have released version 1.1.1, please update so tests will pass